lib/kiev/rack/request_logger.rb

```diff
@@ -70,6 +70,11 @@ def log_rack_exception?(exception)
         !Config.instance.ignored_rack_exceptions.include?(exception.class.name)
       end
 
+      def real_ip(request)
+        # the Rack::Request#ip is broken for ages
+        ActionDispatch::Request.new(request.env).remote_ip
```

Comment:
not sure if `ActionDispatch` is always there - I've seen things like `if defined?(ActionDispatch)` in here (e.g. `lib/kiev/param_filter.rb`).